pkg/faketikv/task.go

```diff
@@ -180,6 +191,12 @@ func (a *removePeer) Step(r *RaftEngine) {
 		for i, peer := range region.GetPeers() {
 			if peer.GetId() == a.peer.GetId() {
 				region.Peers = append(region.Peers[:i], region.Peers[i+1:]...)
+				for j, voter := range region.GetVoters() {
```

Comment:
use `RemoveStorePeer`, otherwise, we need to maintain the code to be consistent in two places.